bolt/src/main/java/com/slack/api/bolt/service/InstallationService.java

```diff
@@ -60,4 +60,12 @@ default String getInstallationGuideText(String enterpriseId, String teamId, Stri
         return null;
     }
 
+    /**
+     * Deletes all installation data for given workspace or organization.
+     */
+    default void deleteAll(String enterpriseId, String teamId) {
```

Comment:
We don't require to implement this method as long as developers don't use the built-in `app_uninstalled` event handler.